config/datasets/poesy.json

```diff
@@ -61,5 +61,6 @@
                 "WIND.V.PHYS":{"shortname":"v","level": [1]}
                 }
             }
-        }
+        },
+    "num_inference_pred_steps": 3
```

Comment:
Is this now madatory to add this parameter in all Poesy config file ? There is no way to make it optional, to not change this file for example ?

Reply:
Yes, you're right, I will take it off. Thank you !